### PR TITLE
feat(e2e-test): Add E2E test cases for cstor csi

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Signed-off-by: nsathyaseelan <sathyaseelan.n@mayadata.io>

- This PR is to Migrating the e2e tests for cstor operators from openebs/e2e-test repo into openebs/cstor-operator repo

### CODE REVIEW

File: e2e-tests/apps/busybox/deployers/run_e2e_test.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            #Application replicas for statefulset application
+          - name: APP_REPLICA
+            value: 'replicas=2'
+            
+          # Use 'deprovision' for app-clean up
+          - name: ACTION
+            value: provision
+
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./apps/busybox/deployers/test.yml -i /etc/ansible/hosts -v; exit 0"]
```
Comment (w3aman) [inline e2e-tests/apps/busybox/deployers/run_e2e_test.yml:67]: check path here
./e2e-tests/apps/*****

File: e2e-tests/apps/busybox/deployers/test.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+          ## Generating the testname for deployment
+        - include_tasks: /utils/fcm/create_testname.yml
+
+          ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
```
Comment (w3aman) [inline e2e-tests/apps/busybox/deployers/test.yml:16]: change it at every place
/update_e2e_result_resource.yml

Comment (w3aman) [reply e2e-tests/apps/busybox/deployers/test.yml:16]: check at every place, i think path will be now /e2e-tests/****

File: e2e-tests/apps/busybox/deployers/run_e2e_test.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    metadata:
+      name: e2e
+      labels:
+        app: busybox-e2e
+    spec:
+      serviceAccountName: e2e
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
```
Comment (w3aman) [inline e2e-tests/apps/busybox/deployers/run_e2e_test.yml:18]: we need different image for this other than ansible-runner

**Review by w3aman — CHANGES_REQUESTED** [review]

**Review by w3aman — APPROVED** [review]
/looks good to me

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Pull request merged**